doc/src/site/sphinx/ml/sw_gam.rst

```diff
@@ -21,26 +31,57 @@ and :ref:`model_details_H2OGAMMOJOModel`.
         .. code:: scala
 
             import ai.h2o.sparkling._
-            import java.net.URI
             val hc = H2OContext.getOrCreate()
 
-        Parse the data using H2O and convert them to Spark Frame
+        Create the frame knots
+
+        .. code:: scala
+
+            val knots1 = Seq(-1.99905699, -0.98143075, 0.02599159, 1.00770987, 1.99942290).toDF()
+            val frameKnots1 = hc.asH2OFrame(knots1)
+            val knots2 = Seq(-1.999821861, -1.005257990, -0.006716042, 1.002197392, 1.999073589).toDF()
+            val frameKnots2 = hc.asH2OFrame(knots2)
+            val knots3 = Seq(-1.999675688, -0.979893796, 0.007573327, 1.011437347, 1.999611676).toDF()
+            val frameKnots3 = hc.asH2OFrame(knots3)
```

Comment:
I would also lines of code that deletes created H2O frames after the model is trained. In future, we could introduce a new parameter `knots` of a dataframe array, which will look after the frame lifecycle automatically.